requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy==1.23.5
+scipy==1.9.3
```

Comment:
Please use a `>=` instead of `==` to avoid dependency conflicts with other packages. Also, make sure that you actually pin the lowest relevant version, i.e. the lowest version that is compatible with the functionality that we depend on.

Reply:
Ok! Now the values are:

`numpy>=1.20.0
scipy>=1.6.0`

Those point to the first versions of the packages that support our older python version that is being tested, which I think is 3.7 . 

If we should use older packages versions please, tell me and I will change it! 😃